### ISSUE TITLE
How to you use the filter feature?

### ISSUE THREAD

**Issue by OCTIAT** [pnedev/compare-plugin#135]
do*

Could someone ELI5 how the filter feature work? This isn't really an issue with the plugin but a question that I'd appreciate if you could answer.

Thanks in advance, and thanks to whoever helped to develop this wonderful plugin.

**Comment by pnedev** [comment]
Hello @OCTIAT ,

I don't understand, what do you mean by "filter feature"? Could you elaborate more?
Thanks.

BR

**Comment by OCTIAT** [comment]
@pnedev For some unknown reason, I thought the filter field in 'Find In Files' had something to do with this plugin. Sorry, I have no idea why I thought that..

**Comment by pnedev** [comment]
I see, OK, thanks.

BR
